client/src/lib/components/sandbox/squad-selection/DraftNewUnits.svelte

```diff
@@ -31,15 +31,49 @@
 			on:keypress={() => {
 				openUnitDraftModal(unit);
 			}}
-			class="border border-slate-300 rounded p-3 col-span-1"
+			class="border border-slate-300 rounded p-3 col-span-1 cursor-pointer"
 		>
 			<p><b>{unit.name}</b></p>
-			<p>Cost: {unit.pointsCost}</p>
-			<p>HP: {unit.maxHealth}, Armor: {unit.armorSaveRoll}</p>
-			<p>Melee: {unit.meleeHitRoll}+/{unit.meleeWoundRoll}+/{unit.meleeArmorPiercing}/{unit.meleeDamage}</p>
-			{#if unit.rangedNumAttacks}
-				<p>Ranged: {unit.rangedHitRoll}+/{unit.rangedWoundRoll}+/{unit.rangedArmorPiercing}/{unit.rangedDamage}</p>
-			{/if}
+			<table class="mx-auto">
+				<tr class="[&>*]:px-[3px]">
+					<th><i class="fa-solid fa-shield" title="Armor Save Roll"></i></th>
+					<th><i class="fa-solid fa-heart" title="Max Health"></i></th>
+					<th><i class="fa-solid fa-circle-dollar" title="Points Cost"></i></th>
+				</tr>
+				<tr class="[&>*]:px-[3px]">
+					<td title="Armor save roll">{unit.armorSaveRoll}+</td>
+					<td title="Max Health">{unit.maxHealth}</td>
+					<td title="Points Cost">{unit.pointsCost}</td>
+				</tr>
+			</table>
+			<table class="mt-4 mx-auto">
+				<tr class="[&>*]:px-[3px]">
+					<th></th>
+					<th><i class="fa-solid fa-hashtag" title="Number of Attacks"></i></th>
+					<th><i class="fa-solid fa-bullseye-arrow" title="Hit Roll"></i></th>
+					<th><i class="fa-solid fa-hand-fist" title="Wound Roll"></i></th>
+					<th><i class="fa-solid fa-shield-slash" title="Armor Piercing"></i></th>
+					<th><i class="fa-solid fa-heart-crack" title="Damage"></i></th>
+				</tr>
+				<tr class="[&>*]:px-[3px]">
```

Comment:
What is `[&>*]:px-[3px`

Reply:
adds `class="px-[3px]"` to all direct children of this tag

Reply:
:0